src/optimization.py

```diff
@@ -3,9 +3,9 @@
 
 
 def maximizeMarginalRate(bounds, A, b, noLog=True):
+    # A is a matrix and b is a column vector
```

Comment:
For this part, I mean what are A and b when the function is called.
I'd like to know how and when to call this function. For example, say A is a matrix of traffic patterns, b is a vector of NF cycles. (Sorry, I don't understand this function.)